runtime/testy/studentAPI.py

```diff
@@ -78,6 +78,9 @@ def __init__(self, toManager, fromManager):
     super().__init__(toManager, fromManager)
     self._createSensorMapping()
     self._coroutines_running = set()
+    self.hibikeEnumerateDevices()
```

Comment:
This will error if hibike isn't spawned yet.
Process spawning order: 
SM -> Hibike -> Student code

putting it in student code is the only way to get the ordering synced up.

We can also put it in hibike. in fact, hibike already does that. This is for additional guarantees in case... hibike break.